src/gasman.c

```diff
@@ -1100,6 +1101,22 @@ static void MaybeMoveBags(void)
 
 #endif
 
+/****************************************************************************
+**
+*F  ReportOutOfMemory(size, reason) . . . . . .  handle running out of memory
+**
+*/
+// ReportOutOfMemory is a macro so the correct filename/line is passed to
```

Comment:
It's a bit jarring to see this mix of comment styles. Why not just turn the `//` into `**` and move the `*/` below, then it fits? Or else, just use `//` for the whole comment.